package.json

```diff
@@ -5,13 +5,13 @@
   "author": "Automattic",
   "scripts": {
     "cm": "git-cz",
-    "semantic-release": "semantic-release",
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "calypso-build",
     "dev": "NODE_ENV=development calypso-build --watch",
     "start": "npm run dev",
     "clean": "rm -rf dist release",
-    "release": "mkdir -p release && zip -r release/newspack-app-shell.zip . -x node_modules/\\* .git/\\* .github/\\* .gitignore .DS_Store"
+    "release:archive": "mkdir -p release && zip -r release/newspack-app-shell.zip . -x node_modules/\\* vendor/\\* src/\\* .git/\\* .github/\\* .gitignore .DS_Store",
+    "release": "npm run build && npm run semantic-release"
```

Comment:
```
npm ERR! missing script: semantic-release
```

Unless I'm missing something, the `semantic-release` script was removed on line 8.

Reply:
Good catch! re-added in 4869459